frontend/src/Pages/ResetPasswordPage.tsx

```diff
@@ -39,17 +44,18 @@ export default function ResetPasswordPage() {
             });
     }, [code, api]);
 
+    const onPasswordStrengthUpdate = useCallback((newStrength: PasswordStrength | undefined) => {
```

Comment:
What's the reason behind wrapping a state setter in useCallback?

Reply:
removed it, I was troubleshooting something and kept it like that, but it's redundant